src/components/calcite-tip-manager/calcite-tip-manager.tsx

```diff
@@ -236,19 +236,29 @@ export class CalciteTipManager {
   }
 
   render() {
-    if (this.total === 0) {
+    const { direction, groupTitle, selectedIndex, textClose, total } = this;
+
+    if (total === 0) {
       return <Host />;
     }
     return (
       <Host onKeydown={this.tipManagerKeyDownHandler}>
         <div tabindex="0">
           <header class={CSS.header}>
-            <h2 class={CSS.heading}>{this.groupTitle}</h2>
-            <calcite-action text={this.textClose} onClick={this.hideTipManager} class={CSS.close}>
+            <h2 key={selectedIndex} class={CSS.heading}>
```

Comment:
Not sure why this is getting the key. `renderPagination` makes that subtree dynamic, so the nodes at that level need a unique key (I would need to test to confirm, but maybe the dynamic node is the only one that needs a key).

Reply:
It needs the key because the textContent needs to be updated for the node and it didn't have a way to know that

Reply:
Follow-up: `key` is being used here to force a re-render of that child. Thanks for the clarification, @driskull!